.travis.yml

```diff
@@ -11,19 +12,35 @@ jobs:
         CXX: clang++
         CC: clang
 
+    #### MAC OS #######
     - os: osx
       osx_image: xcode10.1
       language: generic
       env:
         CXX: clang++
         CC: clang
+
+    #### WINDOWS #######
     - os: osx
       osx_image: xcode10.1
       language: generic
       env:
         CXX: g++
         CC: gcc
 
+    - os: windows
+      language: c++
+      env:
+        - MINCONDA_VERSION="latest"
+        - CMAKE_GENERATOR="Visual Studio 15 2017 Win64"
+        - TESTS_TARGET="RUN_TESTS";
+      install:
+        - choco install --yes miniconda3
+        - source C:/Tools/miniconda3/Scripts/activate
+        - conda config --set always_yes yes --set changeps1 no
+        - conda update -q conda
+        - conda install -c conda-forge hdf5
```

Comment:
do the windows wheels work on a clean windows install, without conda?

Reply:
I haven't trying tackling the wheel problem yet. The goal of this PR is to enable Windows CI. It's not an official Windows support yet. I was planning to start working on the wheels in a subsequent PR.